src/main/java/racingcar/domain/game/RacingGame.java

```diff
@@ -0,0 +1,60 @@
+package racingcar.domain.game;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import racingcar.domain.car.Car;
+import racingcar.domain.car.DefaultCar;
+import racingcar.domain.numbergenerator.DefaultRandomNumberGenerator;
+import racingcar.domain.numbergenerator.RandomNumberGenerator;
+import racingcar.view.RacingGameView;
+
+public class RacingGame implements Game {
+
+    private List<Car> cars;
+    private int attempts;
+
+
+    @Override
+    public void start() {
+        RacingGameView view = new RacingGameView(this);
+        initialize(view);
+        race(view);
+    }
+
+    private void race(RacingGameView view){
```

Comment:
System In으로 입력받는 값은 멤버 변수로 관리하지 않고, 외부에서 원시 값으로 받아도 괜찮을 것 같은데요?

```suggestion
    private void race(int attempts){
```